ha/core/system_health/factory/action_factory.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+from ha.core.system_health.handlers.action_handler import ActionHandler, NodeFruPSUActionHandler, \
+    NodeFruFanActionHandler, NodeSWActionHandler
+from ha.core.system_health.model.health_event import HealthEvent
+from ha.core.event_manager.error import InvalidResourceType, HAActionHandlerError
+
+EVENT_ACTION_HANDLERS_MAPPING = {
+    "node:fru:psu": NodeFruPSUActionHandler,
+    "node:fru:fan": NodeFruFanActionHandler,
+    "node:fru:disk": None,
+    "node:sensor:temperature": None,
+    "node:sensor:voltage": None,
+    "node:os:cpu": None,
+    "node:os:cpu:core": None,
+    "node:os:disk_space": None,
+    "node:os:memory_usage": None,
+    "node:os:memory": None,
+    "node:os:raid_data": None,
+    "node:os:raid_integrity": None,
+    "node:os:disk": None,
+    "node:interface:nw": None,
+    "node:interface:nw:cable": None,
+    "node:interface:sas": None,
+    "node:interface:sas:port": None,
+    "node:sw:os:service": NodeSWActionHandler
```

Comment:
We need to handle server failure in this PI. What is the resource type for a server and where is it's action handler?

Reply:
Done, Added only node action failure handler and mapping of it in the action factory. resource type for server will be only "node"